webapp/ordering/models.py

```diff
@@ -27,8 +27,11 @@ class Meta:
 
     name = models.CharField(max_length=50, unique=True)
     address = models.ForeignKey(Address)
+    contactperson = models.CharField(max_length=50, blank=True)
```

Comment:
Can you change this to hyphenated `contact_person` like the other two fields? (just do search&replace in migration, no need for a second migration).

Otherwise: 👍

Reply:
Good one. done.